test/parallel/test-net-socket-destroy-twice.js

```diff
@@ -23,7 +23,7 @@
 const common = require('../common');
 const net = require('net');
 
-const conn = net.createConnection(common.PORT);
+const conn = net.createConnection(0);
```

Comment:
@Trott I've looked closely at the purpose of this test, it is to trigger an "already destroyed" path. Since there is nothing to connect to it raises an error, on any error destroy socket, on close destroy socket again, alert it has already been destroyed. Do you agree?

The comment has been added.